ratis-assembly/src/main/assembly/bin.xml

```diff
@@ -77,35 +77,35 @@
       <directoryMode>0755</directoryMode>
     </fileSet>
     <!-- Include dev support tools -->
-    <fileSet>
-      <directory>${project.basedir}/../dev-support</directory>
-      <outputDirectory>dev-support</outputDirectory>
-      <fileMode>0644</fileMode>
-      <directoryMode>0755</directoryMode>
-    </fileSet>
-    <fileSet>
-      <directory>${project.basedir}/../ratis-shell/target/</directory>
-      <outputDirectory>ratis-shell/lib/shell</outputDirectory>
-      <fileMode>755</fileMode>
-      <includes>
-        <include>ratis-shell-*-jar-with-dependencies.jar</include>
-      </includes>
-    </fileSet>
-    <fileSet>
-      <directory>${project.basedir}/../ratis-shell/src/main/bin</directory>
-      <outputDirectory>ratis-shell/bin</outputDirectory>
-      <fileMode>755</fileMode>
-    </fileSet>
-    <fileSet>
-      <directory>${project.basedir}/../ratis-shell/src/main/libexec</directory>
-      <outputDirectory>ratis-shell/libexec</outputDirectory>
-      <fileMode>0644</fileMode>
-      <directoryMode>0755</directoryMode>
-    </fileSet>
-    <fileSet>
-      <directory>${project.basedir}/../ratis-shell/src/main/conf</directory>
-      <outputDirectory>ratis-shell/conf</outputDirectory>
-      <fileMode>644</fileMode>
-    </fileSet>
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../dev-support</directory>-->
+<!--      <outputDirectory>dev-support</outputDirectory>-->
+<!--      <fileMode>0644</fileMode>-->
+<!--      <directoryMode>0755</directoryMode>-->
+<!--    </fileSet>-->
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../ratis-shell/target/</directory>-->
+<!--      <outputDirectory>ratis-shell/lib/shell</outputDirectory>-->
+<!--      <fileMode>755</fileMode>-->
+<!--      <includes>-->
+<!--        <include>ratis-shell-*-jar-with-dependencies.jar</include>-->
+<!--      </includes>-->
+<!--    </fileSet>-->
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../ratis-shell/src/main/bin</directory>-->
+<!--      <outputDirectory>ratis-shell/bin</outputDirectory>-->
+<!--      <fileMode>755</fileMode>-->
+<!--    </fileSet>-->
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../ratis-shell/src/main/libexec</directory>-->
+<!--      <outputDirectory>ratis-shell/libexec</outputDirectory>-->
+<!--      <fileMode>0644</fileMode>-->
+<!--      <directoryMode>0755</directoryMode>-->
+<!--    </fileSet>-->
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../ratis-shell/src/main/conf</directory>-->
+<!--      <outputDirectory>ratis-shell/conf</outputDirectory>-->
+<!--      <fileMode>644</fileMode>-->
+<!--    </fileSet>-->
```

Comment:
Why commenting all these lines?  For debugging?

Reply:
Hi @szetszwo, the content in `ratis-shell` are already in the root dir.

Reply:
Close the comment now.